src/Microsoft.Health.Fhir.Shared.Api/Controllers/OperationController.cs

```diff
@@ -0,0 +1,64 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using System.Threading.Tasks;
+using EnsureThat;
+using Hl7.Fhir.Model;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Health.Api.Features.Audit;
+using Microsoft.Health.Fhir.Api.Features.ActionResults;
+using Microsoft.Health.Fhir.Api.Features.Filters;
+using Microsoft.Health.Fhir.Api.Features.Routing;
+using Microsoft.Health.Fhir.Core.Exceptions;
+using Microsoft.Health.Fhir.Core.Features;
+using Microsoft.Health.Fhir.Core.Messages.Operation;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.ValueSets;
+
+namespace Microsoft.Health.Fhir.Api.Controllers
+{
+    [ServiceFilter(typeof(AuditLoggingFilterAttribute))]
+    [ServiceFilter(typeof(OperationOutcomeExceptionFilterAttribute))]
+    [ValidateResourceTypeFilter]
+    [ValidateModelState]
+    public class OperationController : Controller
```

Comment:
>OperationController [](http://example.com/codeflow?start=17&length=19)

I think right now we put every spec defined functionallity into FhirContoller, and everything not aligned with spec in separate controllers (Export, Reindex).

I would move it to FhirController.